verification/cbmc/stubs/hash_table_generators.c

```diff
@@ -32,8 +34,12 @@ void hash_find_string_string_generator(
     const struct aws_hash_table *map,
     const void *key,
     struct aws_hash_element *p_elem) {
-    p_elem->key = ensure_string_is_allocated_nondet_length();
-    p_elem->value = ensure_string_is_allocated_nondet_length();
+    if (p_elem) {
+        p_elem->key = ensure_string_is_allocated_nondet_length();
+        __CPROVER_assume(p_elem->key != NULL);
+        p_elem->value = ensure_string_is_allocated_nondet_length();
+        __CPROVER_assume(p_elem->value != NULL);
+    }
```

Comment:
Same question here.

Reply:
Same as above (except for find element instead of iterate).